test/EndToEndTests/Common/Microsoft.OData.Client.E2E.TestCommon/Samples/Server/SampleModel.cs

```diff
@@ -0,0 +1,36 @@
+﻿//-----------------------------------------------------------------------------
+// <copyright file="SampleModel.cs" company=".NET Foundation">
+//      Copyright (c) .NET Foundation and Contributors. All rights reserved.
+//      See License.txt in the project root for license information.
+// </copyright>
+//------------------------------------------------------------------------------
+
+using EfKey = System.ComponentModel.DataAnnotations.KeyAttribute;

```

Comment:
Is there any specific reason for aliasing `KeyAttribute`? Is it just a legacy thing?

Reply:
It's because the project references `Microsoft.OData.Client` which also has KeyAttribute with the same namespace.